### PR TITLE
nixos/meme-bingo-web: fixed documentation for baseUrl option; updated meme-bingo-web

### PR DESCRIPTION
**Pull request by auroraanna**
## Description of changes

- I noticed that the `<base>` in the documentation for the baseUrl option is interpreted as an HTML element on https://search.nixos.org because it's written in Markdown. This should make it be interpreted as just text from my Markdown experience.
- In some places systemd adds a period to the service description. Since this service description already has a period, systemd makes it a double period. This PR removes the period.
- some hardening a friend suggested, supposedly increases `systemd-analyze security` score

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/modules/services/web-apps/meme-bingo-web.nix [hunk 1]
```diff
@@ -22,7 +22,7 @@ in {
 
       baseUrl = mkOption {
         description = mdDoc ''
-          URL to be used for the HTML <base> element on all HTML routes.
+          URL to be used for the HTML \<base> element on all HTML routes.
```
Comment (donovanglover) [inline nixos/modules/services/web-apps/meme-bingo-web.nix:25]:
```suggestion
          URL to be used for the HTML \<base\> element on all HTML routes.
```

A quick `rg "\\\<"` shows [other](https://github.com/NixOS/nixpkgs/blob/b58b0b0311f083da10dbb936539fe1cb85c1a6fb/nixos/modules/services/networking/ndppd.nix#L111) `mdDoc`s that also escape `<text>` like this.

**Review by donovanglover — CHANGES_REQUESTED** [review]

**Review by donovanglover — APPROVED** [review]
LGTM

**Comment by auroraanna** [comment]
oops, I thought this was merged already

**Review by donovanglover — APPROVED** [review]
LGTM

**Comment by auroraanna** [comment]
`systemd-analyze security` looks pretty good already at 1.1 (OK). The PR didn't change the score.

**Review by donovanglover — APPROVED** [review]

**Comment by donovanglover** [comment]
Result of `nixpkgs-review pr 255383` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme-bingo-web</li>
  </ul>
</details>

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by auroraanna** [comment]
Whoopsy daisy, found a breaking bug and fixed it.

**Comment by donovanglover** [comment]
Result of `nixpkgs-review pr 255383` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme-bingo-web</li>
  </ul>
</details>

**Review by donovanglover — APPROVED** [review]
LGTM. Could be a fun exercise to write some tests upstream

**Comment by auroraanna** [comment]
@donovanglover Yeah, the tests might've caught it, but the test would have needed to include a browser. If the tests are just requesting the URL which the form is also requesting, then it wouldn't have been found. Can you even do tests with displays?

**Comment by auroraanna** [comment]
Figured out how to do headless teasting, actually taking into account the CSP e.g.
Should be in the next release.

**Comment by donovanglover** [comment]
Can you rebase?

**Comment by donovanglover** [comment]
Result of `nixpkgs-review pr 255383` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
    <li>tests.nixos-functions.nixos-test</li>
  </ul>
</details>
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>meme-bingo-web</li>
  </ul>
</details>

**Review by donovanglover — APPROVED** [review]

**Pull request merged**